Sources/ClientRuntime/EventStream/MessageUnmarshaller.swift

```diff
@@ -0,0 +1,17 @@
+//
+// Copyright Amazon.com Inc. or its affiliates.
+// All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+/// Unmarshals a `Message` into a event stream event.
+public protocol MessageUnmarshaller {
```

Comment:
Nit: Similar to an bove comment, imo we should use `Unmarshallable` similar to Decodable/Encodable. Since the conforming object is what is being unmarshalled from an EventStream.Message

```suggestion
public protocol MessageUnmarshallable {
```

Reply:
fixed